nifi-nar-bundles/nifi-standard-services/nifi-hwx-schema-registry-bundle/nifi-hwx-schema-registry-service/src/main/java/org/apache/nifi/schemaregistry/hortonworks/HortonworksSchemaRegistry.java

```diff
@@ -66,12 +66,12 @@
 @CapabilityDescription("Provides a Schema Registry Service that interacts with a Hortonworks Schema Registry, available at https://github.com/hortonworks/registry")
 public class HortonworksSchemaRegistry extends AbstractControllerService implements SchemaRegistry {
     private static final Set<SchemaField> schemaFields = EnumSet.of(SchemaField.SCHEMA_NAME,
-            SchemaField.SCHEMA_BRANCH_NAME,
-            SchemaField.SCHEMA_TEXT,
-            SchemaField.SCHEMA_TEXT_FORMAT,
-            SchemaField.SCHEMA_IDENTIFIER,
-            SchemaField.SCHEMA_VERSION,
-            SchemaField.SCHEMA_VERSION_ID);
+        SchemaField.SCHEMA_BRANCH_NAME,
```

Comment:
@tpalfy Please do not change the indentation / reformat the code if it is not necessary. The line continuation indent should be 2 x normal indent, so I think the original was fine.

Otherwise, the change looks good to me.